app/models/stock.rb

```diff
@@ -0,0 +1,16 @@
+class Stock < ApplicationRecord
+  has_many :user_stocks, dependent: :destroy
+  has_many :users, through: :user_stocks
+
+  validates :name, :ticker, presence: true
+
+  def self.new_lookup(ticker_symbol)
+    client = IEX::Api::Client.new(publishable_token: Rails.application.credentials.iex_client[:sandbox_api_key],
+                                    endpoint: 'https://sandbox.iexapis.com/v1')
+    begin
+      new(ticker: ticker_symbol, name: client.company(ticker_symbol).company_name, last_price: client.price(ticker_symbol))
+    rescue StandardError
```

Comment:
I think you can specify here the exception type/class that you are going to rescue.

Reply:
hi maam, do you mean Raise Exception return e etc?.. that was our initial code and it is failing in rubocop hence we just decided to use this as alternative instead

Reply:
Check first the exception type/class that `iex` is throwing instead of using the `StandardError`.

Reply:
@analyn-cajocson finished resolving all comments!